src/researches/passiveVoice/morphological/determinePassiveSentence.js

```diff
@@ -0,0 +1,48 @@
+const filter = require( "lodash/filter" );
+const getWords = require( "../../../stringProcessing/getWords.js" );
+
+// Verb-form lists per language
+const getPassiveVerbsRussian = require( "../../russian/passiveVoice/participlesShortenedList.js" )().all;
+
+/**
+ * Matches the sentence against passive verbs.
+ *
+ * @param {string} sentence The sentence to match against.
+ * @param {Array} passiveVerbs The array containing passive verb-forms.
+ * @returns {Array} The found passive verbs.
+ */
+let matchPassiveVerbs = function( sentence, passiveVerbs ) {
+	return filter( getWords( sentence ), function( word ) {
+		return passiveVerbs.includes( word.toLocaleLowerCase() );
+	} );
+};
+
+/**
+ * Checks the passed sentences to see if they contain passive verb-forms.
+ *
+ * @param {string} sentence The sentence to match against.
+ * @param {string} language The language of the text.
+ * @returns {Array} The list of encountered passive verbs.
+ */
+let determineSentenceIsPassive = function( sentence, language ) {
+	let passiveVerbs = [];
+
+	switch ( language ) {
+		case "ru":
+			passiveVerbs = getPassiveVerbsRussian;
```

Comment:
Can the Russian passive verbs only be required if this condition is true, instead of always?
In other words, can the `require` be placed inside the `case` statement.

Reply:
Agreed, fixed.

Reply:
Using `require` within a function gives an error in `grunt check`, although otherwise works just fine. Given that all other assessments are written so that they `require` all language-specific word lists in the beginning of the file, not within `case` statement, I have questions to @atimmer. First, is it at all a problem to load potentially big arrays in the beginning of the file, can it indeed slow down our assessments tremendously? And if so, shall we make a general issue out of it and tackle inefficient imports for all assessments at once?

Reply:
It can't. Browserify cannot deal with dynamic require statements. We wil need to fix this for all word lists, because the bundle size is growing rapidly.

Reply:
I created an issue #1463 to address this issue for all assessments that make use of large word lists.